docs/Using-Pantheon/Debugging.md

```diff
@@ -3,10 +3,52 @@ description: Frequently asked questions FAQ and answers for troubleshooting Pant
 
 # Debugging Pantheon
 
-## My command line options are not working as I expected? 
+## Command Line Options Not Working as Expected
 
-Ensure quotes have not been automatically converted to smart quotes or hyphens combined if copying and pasting. 
+Characters such as smart quotes and long (em) hyphens won't work in Pantheon options. Ensure that quotes have
+not been automatically converted to smart quotes or that double hyphens have not been combined into em hyphens.
 
-## How can I monitor node performance and connectivity? 
+## Monitor Node Performance and Connectivity Using the JSON-RPC API
 
-Using the [`debug_metrics`](JSON-RPC-API-Methods#debug_metrics) JSON-RPC API method.  
\ No newline at end of file
+You can monitor node performance using the [`debug_metrics`](../Reference/JSON-RPC-API-Methods.md#debug_metrics)
+JSON-RPC API method.
+
+## Monitor Node Performance Using Third-Party Clients
+
+You can monitor node performance using the [Prometheus](https://prometheus.io/) monitoring and alerting service.
+You can also visualize the collected data using [Grafana](https://grafana.com/).
+
+Use the Pantheon CLI option [`--metrics-enabled`](../Reference/Pantheon-CLI-Syntax.md#metrics-enabled) to allow
```

Comment:
Suggest rewording as 
Set to the `--metrics-enabled` command line option to `true` to enable the [Prometheus](https://prometheus.io/) monitoring service access to Pantheon metrics.

- Given this is Pantheon documentation I think we can assume we're talking about Pantheon command line options.
- I'd be explicit about how to set the option here so they don't have to go from here to the CLI reference just to find out it needs set to true.

Reply:
Done